packages/server/test/api-emotes.js

```diff
@@ -0,0 +1,64 @@
+const { test } = require('ava')
+const { testWithServer, makeAdmin } = require('./_serverUtil')
+const fetch = require('./_fetch')
+const nodeFetch = require('node-fetch')
+
+let portForApiEmoteTests = 27000
```

Comment:
Eh, what was the reasoning for removing `findport`? Sometimes ports can be used and this might make tests fail on particular machines - would be nice if `spawn` picked an open port at least (logic: `port++; if port.open return port else recurse`)

Reply:
See https://github.com/decent-chat/decent/pull/204/commits/93c6d481b6e15bbbdcb59097f9072fb19ea9c68f.

>```
>Unfortunately, openPort doesn't work well when we're dealing with a lot
>of things running concurrently (and in separate processes
>simultaneously). Instead, we just use a base port number constant in
>each test file and increase that for each spawned server. (So, 21000 in
>tests/auth.js, 22000 in tests/middleware.js, 23000 in the next test file
>we make, etc -- these need to be very far from each other because we
>don't want to have conflicts and, as I mentioned, openPort doesn't work.)
>```

I agree with you, though. Want to have a go at making that more clean and avoid used ports? I still think we should keep these ports separated in blocks of [big number, currently 1000] though - as it would turn out, there's sometime between when we decide a port number (e.g. via `openPort`) and when that port is actually used (probably because of databases opening, etc). And that delay becomes much more significant when we're dealing with two test files running literally concurrently.